crates/bevy_app/src/app_builder.rs

```diff
@@ -125,6 +141,31 @@ impl AppBuilder {
         self
     }
 
+    /// Adds a system that runs every time `app.update()` is called by the runner
+    ///
+    /// Systems are the main building block in bevy ECS model. You can define
+    /// normal rust functions, and call `.system()` to make them be bevy systems.
+    ///
+    /// System functions can have parameters, through which one can query and
+    /// mutate bevy ECS states. See bevy book for extra information.
```

Comment:
Nit: `See [The Bevy Book](https://bevyengine.org/learn/book/introduction/) for more information`.